### PR TITLE
Setup Appveyor CI for Windows

### PR DESCRIPTION
**Pull request by dawagner**
See https://ci.appveyor.com/project/dawagner/parameter-framework

Only xmlserializer is built for now.
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/01org/parameter-framework/pull/177%23issuecomment-136807020%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/177%23issuecomment-136812168%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/177%23issuecomment-136819060%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/177%23discussion_r38448259%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/01org/parameter-framework/pull/177%23issuecomment-136807020%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%40OznOg%20%40krocard%20please%20review.%22%2C%20%22created_at%22%3A%20%222015-09-01T17%3A38%3A54Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6430928%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/dawagner%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3A-1%3A%20%22%2C%20%22created_at%22%3A%20%222015-09-01T17%3A56%3A46Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%2C%20%7B%22body%22%3A%20%22%3Ashipit%3A%20%22%2C%20%22created_at%22%3A%20%222015-09-01T18%3A21%3A51Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6430928%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/dawagner%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%20f4360e2aa4d423d9121735f38d0016b56bd3edce%20xmlserializer/CMakeLists.txt%2026%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/01org/parameter-framework/pull/177%23discussion_r38448259%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%60%60%60%20%5Cr%5Cn%23%20careful%20if%20multiple%20libxml2%20are%20found%2C%20check%20will%20fail.%5Cr%5Cn%60%60%60%22%2C%20%22created_at%22%3A%20%222015-09-01T17%3A41%3A42Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20xmlserializer/CMakeLists.txt%3AL34-53%22%7D%7D%2C%20%22approved%22%3A%20%7B%22https%3A//github.com/dawagner%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6430928%3Fv%3D3%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>

<a href='https://github.com/dawagner'></a>

- [ ] <a href='#crh-comment-Pull f4360e2aa4d423d9121735f38d0016b56bd3edce xmlserializer/CMakeLists.txt 26'></a> &nbsp;<b><a href='https://github.com/01org/parameter-framework/pull/177#discussion_r38448259'>File: xmlserializer/CMakeLists.txt:L34-53</a></b>
- <a href='https://github.com/krocard'></a> 
```
# careful if multiple libxml2 are found, check will fail.
```
- [x] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/01org/parameter-framework/pull/177#issuecomment-136807020'>General Comment</a></b>
- <a href='https://github.com/dawagner'></a> @OznOg @krocard please review.
- <a href='https://github.com/krocard'></a> :-1:

<a href='https://www.codereviewhub.com/01org/parameter-framework/pull/177?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/01org/parameter-framework/pull/177?approve=1'></a>&nbsp;<a href='https://www.codereviewhub.com/01org/parameter-framework/pull/177?approve=0'></a>&nbsp;<a href='https://github.com/01org/parameter-framework/pull/177'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by dawagner** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/01org/parameter-framework/pull/177'>Refresh</a>.<a href='#crh-update-none'></a>

**Comment by dawagner** [comment]
@OznOg @krocard please review.

File: xmlserializer/CMakeLists.txt [hunk 1]
```diff
@@ -34,22 +34,19 @@ add_library(xmlserializer SHARED
...
-                           "Please install a version of libxml2 supporting it.")
-    endif()
-endif(NOT LIBXML2_FOUND)
+if (WIN32)
+    set(CMAKE_PREFIX_PATH "${libxml2_prebuilts}")
+endif()
+find_package(LibXml2 REQUIRED)
+# libxml2 has been found, but does it support XInclude ?
+include(CheckLibraryExists)
+check_library_exists("${LIBXML2_LIBRARIES}" xmlXIncludeProcess "" XML2_XINCLUDE_SUPPORT)
```
Comment (krocard) [inline xmlserializer/CMakeLists.txt]: # careful if multiple libxml2 are found, check will fail.

**Comment by krocard** [comment]
:+1:

**Comment by krocard** [comment]
:shipit:

**Comment by krocard** [comment]
:-1:

**Comment by dawagner** [comment]
:shipit:

**Pull request merged**